languages/rust/exercises/concept/structs/.meta/config.json

```
@@ -0,0 +1,8 @@
+{
+    "contributors": [
```

You're actually an author! So this should be:

```suggestion
    "authors": [
```

See [the documentation](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#metaconfigjson).